### PR TITLE
Fix space after comma

### PR DESCRIPTION
**Pull request by abhinavagarwal07**

### CODE REVIEW

File: benchmarks/symbench_ginac.cpp [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // To compile on a debian system you need to install libginac-dev first
 // $ sudo apt-get install libginac-dev
 // Then compile with the following command,
-// $ g++ -std=c++0x -o symbench_ginac -Wl,--no-as-needed `pkg-config --cflags --libs ginac` symbench_ginac.cpp
+// $ g++ -std=c++0x -o symbench_ginac -Wl, --no-as-needed `pkg-config --cflags --libs ginac` symbench_ginac.cpp
```
Comment (isuruf) [inline benchmarks/symbench_ginac.cpp]: There shouldn't be a space here

**Comment by isuruf** [comment]
Keep the formatting of teuchos as it is.

**Comment by abhinavagarwal07** [comment]
Should I squash commits ?

**Comment by certik** [comment]
This is fine with me.

**Comment by abhinavagarwal07** [comment]
@sumith1896 @isuruf
Ping

**Comment by Sumith1896** [comment]
This is fine with me too, +1

**Comment by isuruf** [comment]
@abhinavagarwal07, thanks for the PR

**Pull request merged**